clients/assessment_component/src/assessment/pages/AssessmentPage/AssessmentPage.tsx

```
@@ -0,0 +1,133 @@
+                        return (
+                          <div>
+                            <CompetencyWithAssessmentItem
+                              competency={competency}
+                              assessment={assessment}
+                            />
+                          </div>
+                        )
+                      }
+
+                      return (
+                        <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)